cmd/buildah/push.go

```diff
@@ -78,15 +79,23 @@ var (
 )
 
 func pushCmd(c *cli.Context) error {
-	args := c.Args()
-	if len(args) < 2 {
-		return errors.New("source and destination image IDs must be specified")
-	}
 	if err := parse.ValidateFlags(c, pushFlags); err != nil {
 		return err
 	}
-	src := args[0]
-	destSpec := args[1]
+
+	args := c.Args()
+	var src, destSpec string
+	switch len(args) {
+	case 0:
+		return errors.New("source and destination image IDs must be specified")
```

Comment:
This error is no longer true as you only need to specify a source with this change.  I'm struggling with verbiage, but maybe something like:

"At least a source image ID must be specified"

I'm struggling with trying to work the thought of the source becoming the destination if not specified, but can't come up with concise verbiage.  Perhaps the above is enough?